loomi/regulariza-diagonal-client
# Feature/nucleus page

## Feature
 - Nucleus Page Layout
 - General Dashboard Layout
 - Research Dashboard Layout
 - Change 'area' to 'nucleus' in modals, tooltips...
 - FilterButton, FilterDrawer
 - SearchBar
---

**What I did:**
   
- Created the pages above,
- Add the filter button and filter drawer
- Mock Nucleus, Nucleus Charts and used the url params to change only the name of the nucleus in nucleus detail page and research dashboard


**How to test:**

1. In your browser go to [stage](http://project.netlify.app/) or [localhost](http://localhost:3000/)
2. Check if the component/view looks just like the [mockup](https://xd.adobe.com/) in different screen sizes (width between 1024px and 1920px)
3. Test the things that you think are worth testing, even the ones that are not in this description

![image](https://user-images.githubusercontent.com/32523401/129496298-9a4299b9-7f57-430b-a2e6-0d26d38662a2.png)
![image](https://user-images.githubusercontent.com/32523401/129496307-6e1fc24e-ab32-4efb-9d2c-f483f7470f76.png)
![image](https://user-images.githubusercontent.com/32523401/129496317-9a49ce4f-ed53-4aa3-92cb-15f3a14c9bb8.png)
